### PR TITLE
global for amount of messages displayed in patient file

### PR DESCRIPTION
**Pull request by shachar058**
Added a global so the number of messages in patient file messages widget can be controlled by user.

### CODE REVIEW

File: interface/patient_file/summary/pnotes_fragment.php [hunk 1]
```diff
@@ -147,7 +147,7 @@
         <div class='tab'>
             <?php
             //display all of the notes for the day, as well as others that are active from previous dates, up to a certain number, $N
-            $M = 3; ?>
+            $M = $GLOBALS['num_of_messages_displayed']; ?>
             <br/>
             <?php
```
Comment (bradymiller) [inline interface/patient_file/summary/pnotes_fragment.php]: place text() around $N line 136 and $M in line 221

File: library/globals.inc.php [hunk 2]
```diff
@@ -776,6 +776,14 @@ function gblTimeZones()
             xl('Do not require patient notes to be authorized')
         ),
 
+        'num_of_messages_displayed' => array(
+            xl('Number of Messages to Display in Patient File'),
```
Comment (bradymiller) [inline library/globals.inc.php]: Change to:
Number of Messages Displayed in Patient Summary

File: library/globals.inc.php [hunk 3]
```diff
@@ -776,6 +776,14 @@ function gblTimeZones()
...
+            'num',
+            '3',
+            xl('This is the number of messages that will be displayed in messages widget located in the patient file.')
```
Comment (bradymiller) [inline library/globals.inc.php]: Change to:
This is the number of messages that will be displayed in the messages widget in the patient summary screen.

**Comment by shachar058** [comment]
hi @bradymiller, performed the required changes, waiting for your approval.
thanks!

**Comment by bradymiller** [comment]
hi @shachar058 , Look great. Will bring into the codebase. Thanks for this improvement! -brady

**Pull request merged**